sentry/src/main/java/io/sentry/SentryWrapper.java

```diff
@@ -51,17 +53,18 @@ public static <U> Callable<U> wrapCallable(final @NotNull Callable<U> callable)
    * @return the wrapped {@link Supplier}
```

Comment:
Comment refers to `Hub`, should refer to `Scopes`

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366